hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,219 @@
+                            podSelector: {}
+                            ingress:
+                              - from:
+                                - namespaceSelector:
+                                    matchLabels:
+                                      name: openshift-operator-lifecycle-manager
+                          policyTypes:
+                            - Ingress
+                        - apiVersion: v1
+                          kind: Service
+                          metadata:
+                            name: deployment-validation-operator-metrics
+                            namespace: openshift-deployment-validation-operator
+                            labels:
+                              name: deployment-validation-operator
+                          spec:
+                            ports:
+                            - name: http-metrics
+                              port: 8383
+                              protocol: TCP
+                              targetPort: 8383
+                            selector:
+                              name: deployment-validation-operator
+                        - apiVersion: v1
+                          kind: ConfigMap
+                          metadata:
+                            name: deployment-validation-operator-config
+                            namespace: openshift-deployment-validation-operator
+                            labels:
+                              name: deployment-validation-operator
+                          data:
+                            deployment-validation-operator-config.yaml: |-
+                              checks:
+                                doNotAutoAddDefaults: true
+                                addAllBuiltIn: false
+                                include:
+                                - "host-ipc"
+                                - "host-network"
+                                - "host-pid"
+                                - "non-isolated-pod"
+                                - "pdb-max-unavailable"
+                                - "pdb-min-available"
+                                - "privilege-escalation-container"
+                                - "privileged-container"
+                                - "run-as-non-root"
+                                - "unsafe-sysctls"
+                                - "unset-cpu-requirements"
+                                - "unset-memory-requirements"
+        - apiVersion: apps.open-cluster-management.io/v1
+          kind: PlacementRule
+          metadata:
+            name: placement-deployment-validation-operator
+            namespace: openshift-acm-policies
+          spec:
+            clusterConditions:
+            - status: "True"
+              type: ManagedClusterConditionAvailable
+            clusterSelector:
+              matchExpressions:
+                - key: hypershift.open-cluster-management.io/management-cluster
```

Comment:
This key is also wrong. It should be `hypershift.open-cluster-management.io/hosted-cluster`